### PR TITLE
Various updates to GC/Code Proposal

### PR DESCRIPTION
**Pull request by gabrielcossette**

### CODE REVIEW

File: Content/OfficalGCCodeSupport.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
-# Official GCCode Support
+# Official GC/Code Support
```
Comment (s-laugh) [inline Content/OfficalGCCodeSupport.md:1]: Where has there been a reference to "GC/Code" (with the slash)? Most common I've seen is "GCcode" (lowercase 'c')

**Comment by remyb2canada** [comment]
Why remove the GCCode argument about
"Contribute to GitLab Community Edition (CE). Having a hosted version of GitLab allows the GC to contribute more directly to the GitLab open source project."?

**Comment by gcharest** [comment]
Are we planning on improving it? Let's be realist here. Has the team being
gccode made any changes to it's source code? How is that an added value?

**Comment by remyb2canada** [comment]
@gcharest It's an opportunity that should be considered as beneficial. The GC can contribute back to the OSS community. It probably does not currently because GCCode is not officially supported and does not have the resources needed to bring it to that level.
But if it had those resources, it would have the capability to do so.

**Comment by gcharest** [comment]
BTW, I know it's an added value over a closed source solution but what's
the value proposition for the target audience here.

**Comment by remyb2canada** [comment]
value proposition to the target audience:
- developer audience: get the features you want as part of the baseline
- executive audience: it's a stretch but the new Policy on Service and Digital (4.4.1.6 about CIO of Canada's responsibility on "support innovative practices and technologies, including open-source and open-standard applications") would mean departmental executive would meet their obligation to support the CIO of Canada on that front

**Comment by gcharest** [comment]
If we can frame it to target both groups, I'm in support 😉

File: Content/OfficalGCCodeSupport.md (hunk 1 above)
Comment (gabrielcossette) [reply Content/OfficalGCCodeSupport.md:1]: It's spelled this way on the login page: https://gccode.ssc-spc.gc.ca/users/sign_in

**Comment by gabrielcossette** [comment]
> Why remove the GCCode argument about
> "Contribute to GitLab Community Edition (CE). Having a hosted version of GitLab allows the GC to contribute more directly to the GitLab open source project."?

@remyb2canada I didn't remove it, I just moved it at the end of the list (as I thought the other reasons were more important).

**Pull request merged**